pom.xml

```diff
@@ -3,7 +3,7 @@
     <modelVersion>4.0.0</modelVersion>
     <groupId>com.amihaiemil.web</groupId>
     <artifactId>eo-yaml</artifactId>
-    <version>5.1.8-SNAPSHOT</version><!--rrv-sed-flag-->
+    <version>5.1.8-428-SNAPSHOT</version><!--rrv-sed-flag-->
```

Comment:
@zsandoz Leave this as it was (``5.1.8-SNAPSHOT``) please. No need to change versions manually, rultor will do it for us.

Reply:
Done